test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": ""
```

Comment:
I'm not sure on this one,  I may be biased by my work with previous JS libraries.
Do we have any others with which to measure against?